src/index.ts

```diff
@@ -36,7 +41,94 @@ const listen =
       type: "LISTEN",
     });
 
+
+    interface Grammar {
+      [index: string]: {
+        intent: string;
```

Comment:
by defining the type this way you require your grammar to have intents and entities, but it doesn't, therefore you are getting errors. You should restructure the type (`interface Grammar`) accordingly.